day_7/caesar_cipher.rb

```diff
@@ -0,0 +1,27 @@
+def cipher(input, shift)
+   input_to_ascii_array = input.chars.map {|char| char.ord}
+   shifted = input_to_ascii_array.map {|char| char+shift}
+   shifted.map { |char| char.chr }.join
+end
```

Comment:
WARNING: When you find code on google, call it out and re-write to match your style of programming.